src/styles/colors.ts

```diff
@@ -5,6 +5,7 @@ export const colors = {
   formal: '#efecd6',
   primary: '#1a0259',
   primaryDark: '#0d0030',
+  blue: '#15a1fc',
```

Comment:
Hmmm, this is `primary`

Reply:
No, it's a `blue-500`. But to name it shorter choose these name